src/_playground/documentation/Markdown/Home.md

```diff
@@ -0,0 +1,38 @@
+# Fundamental React
```

Comment:
hrm.. this looks suspiciously like our readme. ;)

Is there anyway we can this content can live in just one location?

alternatively, perhaps different content here?

Reply:
If it's going to be this content, could we use `node-fetch` (or something like it) to get the `https://github.com/SAP/fundamental-react/blob/master/README.md`?  We could extend the functionality of `MarkdownImporter` to do this fetching.

Reply:
@greg-a-smith love this idea - I'll take a shot at implementing it now